content/custom.py

```diff
@@ -119,7 +126,7 @@
 .. |{function}| replace:: ``{function}``
 .. |term-{function}| raw:: html
 
-   <a class="reference internal" href="../quick-reference/#term-{function}"><span class="xref std std-term"><code class="docutils literal notranslate">{function}</code></span></a>
+   <a class="reference internal" href="../quick-reference/index.html#term-{function}"><span class="xref std std-term"><code class="docutils literal notranslate">{function}</code></span></a>
```

Comment:
what's this for? links to glossary seem to work without it

Reply:
My local builds produce working links if we do it this way. Shouldn't matter in principle though...

Reply:
ok thanks, thought i should ask before merging